application/datamanager/features/steps/equity_bars_steps.py

```
@@ -0,0 +1,167 @@
+            "start": start_date,
+            "end": end_date,
+        }
+    }
+    response = requests.get(url, params=params)
+    context.response = response
+    context.start_date = start_date
+    context.end_date = end_date
+
+
+@then("the response should contain equity bars data for the date range")
+def step_impl(context):
```

Ruff (0.11.9)

Redefinition of unused `step_impl` from line 87

(F811)